go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/spf13/cobra v0.0.6
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.4.0
-	github.com/vmware-tanzu/astrolabe v0.3.1-0.20210623224134-91eeed4dcf77
+	github.com/vmware-tanzu/astrolabe v0.3.1-0.20210630004436-d9486ea1aead
```

Comment:
Can you update with this https://github.com/vmware-tanzu/astrolabe/releases/tag/v0.4.0